app/lib/helpers/__tests__/wordWrap.test.ts

```
@@ -0,0 +1,20 @@
+import { wordWrap } from '../helpers';
+
+describe('wordWrap', () => {
+  let txt = '';
+  beforeEach(() => {
+    txt = 'The function wrap a sentence at particular length of characters';
+  });
+  afterEach(() => (txt = ''));
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)